README.md

```diff
@@ -245,10 +251,8 @@ to the Xcode project exported by Unity.
 
 Dependencies for iOS are added by referring to CocoaPods.
 
-   1. Add the unitypackage to your plugin project (assuming you are developing a
-      plugin).  Please read the [Plugin Redistribution](#plugin-redistribution)
-      section when you're considering how to export your package to share with
-      your users.
+   1. Add the `play-services-resolver-1.2.75.0.unitypackage` to your plugin 
```

Comment:
Similar to above, it would be great to make sure users don't miss the redistribution and also should probably avoid having specific version numbers in here.